guide/src/building-and-distribution.md

```diff
@@ -32,11 +32,11 @@ my_main_crate = { path = "..", default-features = false }
 pyo3 = "0.6"
 ```
 
-On linux/mac you might have to change `LD_LIBRARY_PATH` to include libpython, while on windows you might need to set `LIB` to include `pythonxy.lib` (where x and y are major and minor version), which is normally either in the `libs` or `Lib` folder of a python installation.
+On Linux/macOS you might have to change `LD_LIBRARY_PATH` to include libpython, while on windows you might need to set `LIB` to include `pythonxy.lib` (where x and y are major and minor version), which is normally either in the `libs` or `Lib` folder of a Python installation.
 
 ## Distribution
 
-There are two ways to distribute your module as python package: The old [setuptools-rust](https://github.com/PyO3/setuptools-rust) and the new [pyo3-pack](https://github.com/pyo3/pyo3-pack). setuptools-rust needs some configuration files (`setup.py`,  `MANIFEST.in`, `build-wheels.sh`, etc.) and external tools (docker, twine). pyo3-pack doesn't need any configuration files. It can not yet build sdist though ([pyo3/pyo3-pack#2](https://github.com/PyO3/pyo3-pack/issues/2)).
+There are two ways to distribute your module as a Python package: The old is [setuptools-rust](https://github.com/PyO3/setuptools-rust) and the new is [pyo3-pack](https://github.com/pyo3/pyo3-pack). setuptools-rust needs some configuration files (`setup.py`, `MANIFEST.in`, `build-wheels.sh`, etc.) and external tools (docker, twine). pyo3-pack doesn't need any configuration files. It can not yet build sdist though ([pyo3/pyo3-pack#2](https://github.com/PyO3/pyo3-pack/issues/2)).
```

Comment:
I think adding "is" here is a grammatical mistake, it sounds awkward. In the original wording "old" and "new" are simple adjectives describing the two packages, now there is an elided "way" in both that I don't think sounds great.

Maybe we can split the difference by adding commas?

"The old, setuptools-rust, and the new, pyo3-pack."

Reply:
I'll defer to your judgment :)